test/test_codegen.py

```diff
@@ -199,64 +150,108 @@ def test_get_pressure(mechname, usr_np):
+                         [("uiuc", "C2H4"),
+                          ("sandiego", "H2"),
+                          ("uconn32", "C2H2"),
+                          ("gri30", "CH4")])
+@pytest.mark.parametrize("reactor_type",
+                         ["IdealGasReactor", "IdealGasConstPressureReactor"])
 @pytest.mark.parametrize("usr_np", numpy_list)
-def test_get_thermo_properties(mechname, usr_np):
+def test_get_thermo_properties(mechname, fuel, reactor_type, usr_np):
     """This function tests that pyrometheus-generated code
-    computes thermodynamic properties c_p, s_r, h_rt, and k_eq
+    computes thermodynamic properties c_p, s_r, h_rt, g_rt, and k_eq
     correctly by comparing against Cantera"""
     # Create Cantera and pyrometheus objects
-    sol = ct.Solution(f"mechs/{mechname}.yaml")
+    sol = ct.Solution(f"mechs/{mechname}.yaml", "gas")
     ptk_base_cls = pyro.codegen.python.get_thermochem_class(sol)
     ptk = make_jax_pyro_class(ptk_base_cls, usr_np)
 
-    # Loop over temperatures
-    temp = np.linspace(500.0, 3000.0, 10)
-    for t in temp:
+    def error(x):
+        return np.linalg.norm(x, np.inf)
 
-        # Set state in cantera for comparison
-        sol.TP = t, ct.one_atm
+    time = 0.0
+    dt = 1e-6
 
-        # Get properties from pyrometheus and compare to Cantera
-        cp_pm = ptk.get_species_specific_heats_r(t)
-        cp_err = np.linalg.norm(cp_pm - sol.standard_cp_R, np.inf)
-        print(f"cp_pm = {cp_pm}")
-        print(f"cnt_cp = {sol.standard_cp_R}")
-        assert cp_err < 1.0e-13
-
-        s_pm = ptk.get_species_entropies_r(t)
-        s_err = np.linalg.norm(s_pm - sol.standard_entropies_R, np.inf)
-        print(f"s_pm = {s_pm}")
-        print(f"cnt_s = {sol.standard_entropies_R}")
-        assert s_err < 1.0e-13
-
-        h_pm = ptk.get_species_enthalpies_rt(t)
-        h_err = np.linalg.norm(h_pm - sol.standard_enthalpies_RT, np.inf)
-        print(f"h_pm = {h_pm}")
-        print(f"cnt_h = {sol.standard_enthalpies_RT}")
-        assert h_err < 1.0e-13
-
-        keq_pm1 = ptk.get_equilibrium_constants(t)
-        print(f"keq1 = {keq_pm1}")
-        keq_pm = 1.0 / np.exp(ptk.get_equilibrium_constants(t))
-        keq_ct = sol.equilibrium_constants
+    gas_const = ptk.gas_constant
 
-        print(f"keq_pm = {keq_pm}")
-        print(f"keq_cnt = {keq_ct}")
-        print(f"temperature = {t}")
-        # xclude meaningless check on equilibrium constants for irreversible reaction
-        for i, reaction in enumerate(sol.reactions()):
-            if reaction.reversible:
-                keq_err = np.abs((keq_pm[i] - keq_ct[i]) / keq_ct[i])
-                print(f"keq_err = {keq_err}")
-                assert keq_err < 1.0e-13
-        # keq_pm_test = keq_pm[1:]
-        # keq_ct_test = keq_ct[1:]
-        # keq_err = np.linalg.norm((keq_pm_test - keq_ct_test) / keq_ct_test, np.inf)
-        # assert keq_err < 1.0e-13
+    oxidizer = {"O2": 1.0, "N2": 3.76}
```

Comment:
Why are we excluding this from the test?

Reply:
Just making it from a temperature loop to a reactor case. You will notice that the same properties are being tested below. Lemme know if I skipped any.

Reply:
But are we testing the thermo properties of individual species in the reactor case? The deleted portion tests the individual polynomials, which are (except for s maybe) only functions of T.